src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4031,28 +4031,28 @@
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * RECORD_DEF
-     * |--MODIFIERS
-     * |   `--LITERAL_PUBLIC (public)
-     * |--LITERAL_RECORD (record)
-     * |--IDENT (myRecord)
-     * |--LPAREN (()
-     * |--RECORD_COMPONENTS
-     * |   |--RECORD_COMPONENT_DEF
-     * |   |   |--ANNOTATIONS
-     * |   |   |--TYPE
-     * |   |   |   `--IDENT (Comp)
-     * |   |   `--IDENT (x)
-     * |   |--COMMA (,)
-     * |   `--RECORD_COMPONENT_DEF
-     * |       |--ANNOTATIONS
-     * |       |--TYPE
-     * |       |   `--IDENT (Comp)
-     * |       `--IDENT (y)
-     * |--RPAREN ())
-     * `--OBJBLOCK
-     *      |--LCURLY ({)
-     *       `--RCURLY (})
+     * |--RECORD_DEF -&gt; RECORD_DEF
```

Comment:
please remove first `|` on each line. It leads to nowhere.

Reply:
Done.

Reply:
@pbludov @romani shouldn't it be like in [here](https://github.com/checkstyle/checkstyle/pull/9378/files), because I suggest students to have this symbol in each row